pages/project-details/[id]/index.js

```diff
@@ -19,17 +24,68 @@ export default function ProjectDetails({
     );
   }
 
-  const { name, description, sketch } = currentProject;
+  function handleDeleteImageProjects(id) {
+    setProjects(
+      projects.map((project) =>
+        project.image.id === id
+          ? {
+              ...project,
+              image: [project.image].filter((image) => image.id !== id),
+            }
+          : project
+      )
+    );
+  }
+
+  async function handleImageChangeProjects(event) {
+    event.preventDefault();
+
+    const formData = new FormData(event.target);
+
+    setStatusUpload("Loading...");
+
+    const response = await fetch("/api/upload", {
+      method: "post",
```

Comment:
"post" in lowercase! it seems to work and i am not shure if that is ok or if this is better to change